networks/meld/tokens/0x333002599c1bad8d3861baa959d0f9e2a4d57c06.json

```diff
@@ -7,8 +7,8 @@
   "tokenAddress": "0x333002599c1bad8d3861baa959d0f9e2a4d57c06",
   "tokenColorTheme": 4,
   "tokenIcon": "/static/tokens/USDT.svg",
-  "tokenName": "Tether USD",
-  "tokenTicker": "USDT",
+  "tokenName": "Chainport USDT",
+  "tokenTicker": "wUSDT",
```

Comment:
Please discard this change.

Reply:
May I ask why? This was requested by @kenblaue 
![Uploading Screenshot_20241029_121829.png…]()

Reply:
Hi, the team checked and confirmed that this is the actual Chainport USDT: https://meldscan.io/token/0xEf3B8512196Ab65F8603b82D1FA5a29bb5ADFeD0
Please revert this change and update that address instead, thanks.